tests/phpunit/unit-tests/class-llms-test-staging.php

```diff
@@ -137,6 +137,9 @@ public function test_clone_detected() {
 		$this->assertTrue( LLMS_Admin_Notices::has_notice( 'maybe-staging' ) );
 		LLMS_Admin_Notices::delete_notice( 'maybe-staging' );
 
+		// Return to front.
+		set_current_screen( 'front' );
```

Comment:
@eri-trabiccolo here's the line... :rage:

Reply:
@thomasplevy yeah... I actually was running some tests, and found the same :D <3